sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -49,6 +50,7 @@ import {
 } from "./core/models";
 import { KeyVaultClient } from "./core/keyVaultClient";
 import { ProxyOptions, RetryOptions } from "./core";
+import { TracerProxy, Span, SupportedPlugins } from "@azure/core-tracing";
```

Comment:
You should be able to get these from core-http as I believe they are re-exported from there. There’s some weird behavior with our Singleton pattern (due to node module resolution and the sum links rush uses) that breaks the Singleton pattern if you pull directly from core-tracing for now (only when using rush).

Reply:
done!